csrc/ir_internal_nodes.h

```diff
@@ -1946,7 +1946,7 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
 
  private:
   const std::vector<IterDomain*> root_domain_;
-  std::vector<IterDomain*> domain_;
+  std::vector<IterDomain*> leaf_domain_;
```

Comment:
What's the motivation of changing the name? 

"leaf" does sound better to me, so I'm fine with the change.

Reply:
Because some of us have been saying "leaf domain" for a while, but we have never formally named it. On the other hand, we also say "domain domain", which is very confusing...